api/pkg/db/model/model.go

```diff
@@ -76,4 +76,13 @@ type (
 		Login string `gorm:"not null;unique"`
 		Name  string `gorm:"not null;default:null"`
 	}
+
+	UserResourceRating struct {
```

Comment:
Don't we need any not null constraints to any of the fields here ??

Reply:
yeah rating can have a not null constraint